test-files/ui/features/step_definitions/custom_steps.rb

```diff
@@ -840,4 +840,24 @@
 
 Then(/^I should not see the "([^"]*)" on the page$/) do |input| 
   el = page.should have_no_css(input, :visible => true)
-end
\ No newline at end of file
+end
+
+# for invalid features
+Then(/^I should see an invalid input warning for "([^"]*)"/) do |input|
+  el = find(input)
+  el[:class].include?('invalid-input').should eq true
+end
+
```

Comment:
I would say this is the perfect place to combine these steps using a "negate" pattern.  But seeing how long it took to get jenkins jobs to pass we will let it slide.